samples/outlook/preview-apis-signature/work-with-client-signatures.yaml

```diff
@@ -0,0 +1,115 @@
+id: outlook-signature-work-with-client-signatures
+name: Work with Client Signatures
+description: 'Check if the client signature is enabled, disable the client signature, get the compose type, and set a signature.'
+host: OUTLOOK
+api_set:
+    Mailbox: '1.1'
+script:
+    content: |
+        $("#isClientSignatureEnabled").click(isClientSignatureEnabled);
+        $("#disableClientSignature").click(disableClientSignature);
+        $("#getComposeType").click(getComposeType);
+        $("#setSignature").click(setSignature);
+
+        function isClientSignatureEnabled() {
+          // Check if the client signature is currently enabled
+          Office.context.mailbox.item.isClientSignatureEnabledAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log("isClientSignatureEnabledAsync succeeded with result: " + asyncResult.value);
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+
+        function disableClientSignature() {
+          // Disable the client signature
+          Office.context.mailbox.item.disableClientSignatureAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log("disableClientSignatureAsync succeeded");
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+
+        function getComposeType() {
+          // Get the compose type of the current item (compose only)
+          Office.context.mailbox.item.getComposeTypeAsync(function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log(
+                "getComposeTypeAsync succeeded with composeType: " +
+                  asyncResult.value.composeType +
+                  " and coercionType: " +
+                  asyncResult.value.coercionType
+              );
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+
+        function setSignature() {
+          // Set the signature for the current item
+          var signature = $("#signature").val();
+          console.log(`Setting signature to "${signature}".`);
+          Office.context.mailbox.item.body.setSignatureAsync(signature, function(asyncResult) {
+            if (asyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log("setSignatureAsync succeeded");
+            } else {
+              console.error(asyncResult.error);
+            }
+          });
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to check if the client signature is enabled, disable the client signature, get the compose type, and set the signature.</p>
+        </section>
```

Comment:
Insert a statement here to indicate required mode e.g. `<p><b>Required mode</b>: Compose</p>`